workflow-manager/main.go

```diff
@@ -76,29 +77,42 @@ var (
 
 func fail(format string, args ...interface{}) {
 	workflowManagerLastFailure.SetToCurrentTime()
-	log.Fatalf(format, args...)
+	log.Fatal().Msgf(format, args...)
+}
+
+func prepareLogger() {
+	zerolog.LevelFieldName = "severity"
+	zerolog.TimestampFieldName = "timestamp"
+	zerolog.TimeFieldFormat = time.RFC3339Nano
+	zerolog.SetGlobalLevel(zerolog.TraceLevel)
 }
 
 func main() {
+	prepareLogger()
 	startTime := time.Now()
-	log.Printf("starting %s version %s. Args: %s", os.Args[0], BuildInfo, os.Args[1:])
+	log.Info().
+		Str("app", os.Args[0]).
+		Str("version", BuildInfo).
+		Str("Args", strings.Join(os.Args[1:], ",")).
+		Msgf("starting %s version %s. Args: %s", os.Args[0], BuildInfo, os.Args[1:])
```

Comment:
There are several places where values get logged in fields and then also in the message format string. Is that intentional?

Reply:
I initially removed them but then brought it back so with a quick glance to the logs it's visible. I did not do this to more complicated structures that we might not necessarily care about seeing at a quick glance to the logs:

e.g. 

kept it for this case: https://github.com/abetterinternet/prio-server/pull/413/files#diff-58d7389d6ae70e2e9725b328431cd15b6b70b2e9908ce8f5d9b55c84b3f1e3d7R364

removed it for this case:  https://github.com/abetterinternet/prio-server/pull/413/files#diff-58d7389d6ae70e2e9725b328431cd15b6b70b2e9908ce8f5d9b55c84b3f1e3d7R519

It's a very subjective decision.

Reply:
We can be consistent and only do structured if that makes more sense. Your call!

Reply:
I think that over time, as we operate the service, we'll learn more about what's annoying and what's helpful and can always refine these log statements.